### PR TITLE
NO-JIRA: add baremetal recovery tests

### PR DESCRIPTION
**Pull request by tjungblu**

### CODE REVIEW

File: ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-release-4.18.yaml [hunk 1]
```diff
@@ -238,6 +238,31 @@ tests:
...
     workflow: openshift-e2e-cert-rotation-shutdown
+- as: e2e-metal-assisted-three-node-recovery
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        OPENSHIFT_VERSION=4.18
+        WORKERS_COUNT=1
+        TEST_FUNC=test_ctlplane_scaleup
```
Comment (hasbro17) [inline ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-release-4.18.yaml:249]: 3 node is the default for assisted-framework so I think we can entirely remove the part for scaling-up extra workers post-install.
```suggestion
```

Comment (tjungblu) [reply ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-release-4.18.yaml]: removed, thanks!

**Comment by hasbro17** [comment]
/lgtm

Just FYI the failing invariant tests all fail on the same timeout e.g:
```
[sig-scheduling][Early] The openshift-authentication pods [apigroup:oauth.openshift.io] should be scheduled on different nodes [Skipped:SingleReplicaTopology] [Suite:openshift/conformance/parallel]
  github.com/openshift/origin/test/extended/scheduling/pods.go:146
    STEP: Creating a kubernetes client @ 10/07/24 13:47:51.387
  I1007 13:47:51.569882 259 client.go:275] configPath is now "/tmp/configfile669231960"
  I1007 13:47:51.569922 259 client.go:324] The user is now "e2e-test-scheduling-pod-check-z2mf5-user"
  I1007 13:47:51.569935 259 client.go:326] Creating project "e2e-test-scheduling-pod-check-z2mf5"
  I1007 13:47:51.998981 259 client.go:334] Waiting on permissions in project "e2e-test-scheduling-pod-check-z2mf5" ...
  I1007 13:47:52.162255 259 client.go:363] DeploymentConfig capability is enabled, adding 'deployer' SA to the list of default SAs
  I1007 13:47:52.176020 259 client.go:378] Waiting for ServiceAccount "default" to be provisioned...
  I1007 13:47:52.300842 259 framework.go:1193] Waiting for service account "default" secrets () to include dockercfg ...
  I1007 13:47:52.393287 259 framework.go:1193] Waiting for service account "default" secrets () to include dockercfg ...
  I1007 13:47:52.492931 259 framework.go:1193] Waiting for service account "default" secrets () to include dockercfg ...
```

I would have thought they would be fixed in 4.18 by 
https://github.com/openshift/origin/pull/29077
https://issues.redhat.com/browse/OCPBUGS-41550

I think there's something else that needs to be fixed here in the test framework client setup
https://github.com/openshift/origin/blob/master/test/extended/util/client.go#L351-L375

But that's not for this PR.

**Pull request closed without merging**